### PR TITLE
Update gcloudsdk.nuspec

### PR DESCRIPTION
**Pull request by ronca71**
Added python3 dependency

### CODE REVIEW

File: gcloudsdk/gcloudsdk.nuspec [hunk 1]
```diff
@@ -6,20 +6,24 @@
...
     <summary>SDK for Google Cloud</summary>
     <description>
 The Google Cloud SDK is a set of tools for Google Cloud Platform. It contains gcloud, gsutil, and bq, which you can use to access Google Compute Engine, Google Cloud Storage, Google BigQuery, and other products and services from the command-line. You can run these tools interactively or in your automated scripts.
 
 This package always installs the latest version of the Google Cloud SDK from Google. It first retrieves the most current installer, which then fetches the latest version of the SDK and installs it for all users in the default installation path. Updates can be performed through the gcloud cli.
     </description>
+     <dependencies>
++      <dependency id="python3" />
++      <dependency id="chocolatey-core.extension" version="1.1.0" />
++    </dependencies>
```
Comment (IvanaGyro) [inline gcloudsdk/gcloudsdk.nuspec]: Here are some redundant '+' signs.
```suggestion
       <dependency id="python3" />
       <dependency id="chocolatey-core.extension" version="1.1.0" />
     </dependencies>
```

**Review by IvanaGyro — CHANGES_REQUESTED** [review]
The version of 20200527 has not been released before, so we can stay in `0.0.0.20200527` to make the version consistent to the signing time of the executable installer.

I tested this package in [Chocolatey Testing Environment](https://github.com/chocolatey-community/chocolatey-test-environment). It is successfully installed when I tested with the inline provisioner, but I got exit code 2 when I tested with running the command in the box. I had to reopen the shell (or enter `refreshenv`) and to run the install command again to make the installation procees successful. This situation may relate to https://github.com/chocolatey/choco/issues/439. But it seems that the issue was solved years ago. Does we have to concern the user may get exit code 2 if they install this package with the command line without python3 preinstalled?

**Comment by ronca71** [comment]
I suppose "chocolatey-core.extension" dependency is useless. Verified package installation in the Choco Test Env without it and all was completed succesfully.

**Comment by IvanaGyro** [comment]
> 
> 
> I suppose "chocolatey-core.extension" dependency is useless. Verified package installation in the Choco Test Env without it and all was completed succesfully.

Yes. chocolatey-core.extension is not necessary. Would you remove it before merging?

**Comment by ronca71** [comment]
- removed choco core extension as dependency
- aligned version to 20200527
- added a comment in chocolateyinstall.ps1 to warn about Python 3.x required, though I expect users should never run PS script manually

File: gcloudsdk/gcloudsdk.nuspec [hunk 2]
```diff
@@ -6,20 +6,23 @@
   <metadata>
     <id>gcloudsdk</id>
     <!-- Use the signing time of the executable file as the version. -->
-    <version>0.0.0.20200527</version>
+    <version>0.0.0.20205027</version>
```
Comment (IvanaGyro) [inline gcloudsdk/gcloudsdk.nuspec:9]:
```suggestion
    <version>0.0.0.202005027</version>
```
Missed a zero.

Comment (IvanaGyro) [inline gcloudsdk/gcloudsdk.nuspec]:
```suggestion
    <version>0.0.0.20200527</version>
```
Here is a typo.

**Review by IvanaGyro — APPROVED** [review]

**Comment by IvanaGyro** [comment]
@gpoul Could you grant us permission to merge PR?

**Pull request merged**